app/controllers/application_controller.rb

```diff
@@ -67,6 +67,7 @@ def analytics
         request: request,
         sp: current_sp&.issuer,
         session: session,
+        user_session: user_session,
```

Comment:
`user_session` is part of the session already passed in above. Is there a way to re-use that?

Reply:
I was digging around trying to figure where warden stores `user_session` but did not have any luck.

Reply:
(I'll probably take another crack at it today)

Reply:
Try this
https://github.com/18F/identity-idp/blob/6682a3eb793b307697b87be10f372fb08780476b/app/services/out_of_band_session_accessor.rb#L44